.meta/releases/0.7.0.toml

```diff
@@ -0,0 +1,358 @@
+{sha = "379a76f0ebbb6e838008237a729b6c901b304c2c", message = "feat(gcp_pubsub sink): Use custom DNS resolver (#1451)", author = "Bruce Guenter", date = 2019-12-27T16:13:41Z, files_count = 1, insertions_count = 10, deletions_count = 7},
+{sha = "3fe52dd153d453a7c1d3f488157c73de4850a67d", message = "chore(website): Fix scrolling issue when mobile slide out nav closes", author = "binarylogic", date = 2019-12-28T10:59:44Z, files_count = 2, insertions_count = 27, deletions_count = 2},
+{sha = "a4e47e2dd090e4121b6051d7f372821a6aab4b3d", message = "chore(website): Fix broken release notes links, closes #1452", author = "binarylogic", date = 2019-12-28T12:26:49Z, files_count = 2, insertions_count = 7, deletions_count = 7},
+{sha = "265f5bdd62ddf5491626aa7118f81cc36e47fa82", message = "chore(website): Fix badge spacing on changelog", author = "binarylogic", date = 2019-12-28T13:23:27Z, files_count = 1, deletions_count = 1},
+{sha = "ba50491c5fb020c50326ee9ba6a6e9b44b900016", message = "chore(website): Allow containers to be flush", author = "binarylogic", date = 2019-12-28T13:44:18Z, files_count = 2, insertions_count = 6, deletions_count = 1},
+{sha = "45e5743ee187a0c9933decea2c2937698a5b41b2", message = "chore: Fix `make check-fmt`", author = "Alexander Rodin", date = 2019-12-29T12:17:50Z, files_count = 1, insertions_count = 1, deletions_count = 1},
+{sha = "8a7ffc59f58242a6a2ecbd41aa6a62961da99752", message = "fix(buffers): Don't ack with zero items (#1465)", author = "Lucio Franco", date = 2019-12-31T12:06:29Z, files_count = 1, insertions_count = 29, deletions_count = 7},
+{sha = "4367ccd73b1151dfe92730c1a016334a7b9fc6b7", message = "chore(website): Update Vector description", author = "binarylogic", date = 2019-12-31T13:02:47Z, files_count = 12, insertions_count = 94, deletions_count = 103},
+{sha = "5a944a5dd0b64592d17383949dd776e8cd9fb8e6", message = "docs: Call out event", author = "binarylogic", date = 2019-12-31T13:03:54Z, files_count = 2, insertions_count = 2, deletions_count = 2},
+{sha = "47d6d740d38b735c63c496ac1932ceb0da6f4129", message = "docs: Ensure all wildcard examples are shown", author = "binarylogic", date = 2020-01-03T00:19:22Z, files_count = 64, insertions_count = 255, deletions_count = 178},
+{sha = "110175a740531bf24324d431b0b71adf18c01948", message = "chore: Fix links in `CONTRIBUTING.md` (#1475)", author = "Lucio Franco", date = 2020-01-03T14:40:01Z, files_count = 1, insertions_count = 1, deletions_count = 1},
+{sha = "4700b8fac37586f6b6649b88d3eb074b8530690c", message = "enhancement(kubernetes source): Parse additional metadata (#1249)", author = "Kruno Tomola Fabro", date = 2020-01-03T21:05:45Z, files_count = 3, insertions_count = 234, deletions_count = 107},
+{sha = "daa765f407766b54f1469bb5d6dc1c7c9c272ea0", message = "feat(new sink)!: Move existing 'tcp' sink into 'socket' sink. (#1212) (#1404)", author = "Spencer T Brody", date = 2020-01-03T19:42:26Z, files_count = 21, insertions_count = 915, deletions_count = 263},
```

Comment:
Can we consolidate this message to a single PR? Currently it references 2 which is confusing:

https://share.getcloudapp.com/DOuvJvmx